plugins/aws/ecs_helper.go

```diff
@@ -337,6 +337,9 @@ func findHostPort(
 	}
 	if carn != "" {
 		viable := []int{}
+		if len(c.NetworkInterfaces) > 0 {
```

Comment:
Are we sure that NetworkInterfaces would be always defined? Even with Host & Bridge networking?

Reply:
Yup, both have cases have NetworkInterfaces.
I've tested the build on Bridge an awsvpc.